app.js

```diff
@@ -5,7 +5,7 @@ const path = require('path')
 const open = require('open')
 const port = process.env.port || 5000
 
-const tilledSecretApiKey = 'sk_SECRET_KEY_HERE'
+const tilledSecretApiKey = 'sk_14blwMvv8p1zfA7vF1MhANksnECiPsYM2FIWqS6iNiDf0rhD0I4GFARSkejTlodLcBhBTJ5l9qol2GYs435yYHvvDviXG0VMe0lX'
```

Comment:
don't commit the secret key. You'll need to do a new branch without the secret key in here.